skrub/_datetime_encoder.py

```diff
@@ -38,7 +35,8 @@
 
 
 class DatetimeEncoder(BaseEstimator, TransformerMixin):
-    """Transforms each datetime column into several numeric columns for temporal features (e.g year, month, day...).
+    """Transform each datetime column into several numeric columns
```

Comment:
Antislash "\\" performs a visual line break without an actual line break in the source code, so the header is still considered a single line.
```suggestion
    """Transform each datetime column into several numeric columns \
```